src/lib/pages/backoffice/Document/DocumentEditor/DocumentEditor.js

```diff
@@ -37,6 +36,10 @@ export class DocumentEditor extends Component {
     this.cancellableAttachDocument && this.cancellableAttachDocument.cancel();
   }
 
+  get userIsEditing() {
+    return !!this.props.match.params.documentPid;
```

Comment:
Very minor: I think you have a linter issue here, shouldn't it be:
```
const {
      match: {
        params: { documentPid },
      },
} = this.props;
...
```
Did you install Prettier?

Reply:
it does complain about it but the code actually does not work if written that way which is interesting.

Reply:
ok no prob! Thanks for checking :)